test_util/test_installer_ccm.py

```diff
@@ -324,8 +324,6 @@ def establish_host_connectivity():
             vpc.delete()
     else:
         log.info("Test failed! VPC will remain for debugging 1 hour from instantiation")
-    if options.ci_flags:
```

Comment:
So, why can't we disable test failure on non-zero exit code? My issue with suppressing the error code like this is that when the test **setup** fails, the test doesn't return non-zero nor does the smart CI pick up a failing test.

Thus, in the current state, if you truly mess up the test so bad that the setup can't run the check will show green.

I'm fine with breaking this out into a different PR, but the way it is now needs some refinement.